crates/nu-cli/src/reedline_config.rs

```diff
@@ -651,13 +651,14 @@ fn add_parsed_keybinding(
             let pos1 = char_iter.next();
             let pos2 = char_iter.next();
 
-            let char = match (pos1, pos2) {
-                (Some(char), None) => Ok(char),
-                _ => Err(ShellError::UnsupportedConfigValue(
+            let char = if let (Some(char), None) = (pos1, pos2) {
+                Ok(char)
+            } else {
+                Err(ShellError::UnsupportedConfigValue(
```

Comment:
Here the wrapping in `Ok` reads a bit weird as the `Err` is immediately bubbled up on L602 with a `?;`

Reply:
Good point.
Fixed.